modules/ai/site-planner-connect/module.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Elementor\Modules\Ai\SitePlannerConnect;
+
+defined( 'ABSPATH' ) || exit;
+
+class Module {
+	const NOT_TRANSLATED_APP_NAME = 'Site Planner';
+	const PLANNER_ORIGIN = 'https://planner.elementor.com';
+	const HIDDEN_PAGE_SLUG = null;
+
+	public function __construct() {
+		add_action( 'rest_api_init', [ $this, 'on_rest_init' ] );
+		add_action( 'admin_menu', [ $this, 'register_menu_page' ], 100 );
+		add_filter('rest_prepare_application_password', function ( $response, $item, $request ) {
```

Comment:
```suggestion
		add_filter( 'rest_prepare_application_password', function ( $response, $item, $request ) {
```

Reply:
I don't know why the linter and fixer didn't handle it. I will go over all files.